aodndata/soop/soop_co2_nrt_nc_generator.py

```diff
@@ -40,16 +42,11 @@
 VALID_PROJECT = ['IMOS', 'FutureReefMap', 'SOOP-CO2_RT']
 INPUT_RT_PARAMETERS = {'Type', 'PcDate', 'PcTime', 'GpsShipLatitude',
                        'GpsShipLongitude', 'EquTemp', 'CO2StdValue',
-                       'CO2um_m', 'H2Omm_m', 'DryBoxDruckPress', 'EquPress',
-                       'EquH2OFlow', 'LicorFlow', 'IntakeShipTemp', 'MetTrueWindSpKts',
-                       'MetTrueWindDir', 'AtmSeaLevelPress'}
-AA_SPECIFIC_INPUT_PARAMS = {'SBE45Flow', 'TsgSbe45Temp',
-                            'TsgSbe45Salinity'}
-IN_SPECIFIC_INPUT_PARAMS = {'TsgShipTemp',
-                            'TsgShipSalinity', 'TsgShipFlow', 'LabMainSwFlow'}
-
+                       'Li7815_CO2_avg', 'Li7815_H2O_avg', 'DryBoxDruckPress', 'EquPress',
+                       'EquH2OFlow', 'LicorFlow', 'Intake_T1','Intake_T2', 'MetTrueWindSpKts',
```

Comment:
`Intake_T1` and `Intake_T2` still don't appear in the mapping, shall they be included? They're meant to be mapped with `TEMP_1_raw` and `TEMP_2_raw` further down